roverrobotics_driver/src/roverrobotics_ros_driver.cpp

```diff
@@ -72,25 +72,27 @@ RobotDriver::RobotDriver(ros::NodeHandle *nh) {
     ROS_INFO("no 'traction_factor' set; using the default value: '0'");
     odom_traction_factor_ = 0;
   }
-  if (!ros::param::get("angular_a_coef", angular_scaling_params_.a_coef)){
+  if (!ros::param::get("angular_a_coef", angular_scaling_params_.a_coef)) {
     ROS_INFO("no 'angular_a_coef' set; using the default value: 0");
-    angular_scaling_params_.a_coef=0;
+    angular_scaling_params_.a_coef = 0;
   }
-  if (!ros::param::get("angular_b_coef", angular_scaling_params_.b_coef)){
+  if (!ros::param::get("angular_b_coef", angular_scaling_params_.b_coef)) {
     ROS_INFO("no 'angular_b_coef' set; using the default value: 0");
-    angular_scaling_params_.b_coef=1;
+    angular_scaling_params_.b_coef = 1;
```

Comment:
This looks wrong. By default, there should be no angular scaling (a=0, b=0, c=1). The error here is on coefficient b.